model/riscv_insts_svinval.sail

```diff
@@ -12,9 +12,7 @@ mapping clause encdec =
       SINVAL_VMA(rs1, rs2)                                                  if haveSvinval()
   <-> 0b0001011 @ rs2 : regidx @ rs1 : regidx @ 0b000 @ 0b00000 @ 0b1110011 if haveSvinval()
 
-function clause execute SINVAL_VMA(rs1, rs2) = {
-  execute(SFENCE_VMA(rs1, rs2))
-}
+function clause execute SINVAL_VMA(rs1, rs2) = { execute(SFENCE_VMA(rs1, rs2)) }
```

Comment:
I wonder if there should be mandatory wrapping after the 
'{' for function clauses that use `= {` (or a style option).

Reply:
I guess it's possible

But what about `let a = {{{{ 1 }}}}`?  Do you want make it to `let a = 1` also?

Reply:
I wasn't suggesting to remove any braces just to have a rule that if you have
`function clause execute name(args) = { body }` it inserts a newline after the `{` to format it as
```
function clause execute name(args) = {
  body
}
```
If you don't want the line break just write it as `function clause execute name(args) = body`. Does that make sense or am I missing something?

Reply:
Sorry I just read it wrong. Done